advanced_functionality/multi_model_bring_your_own/multi_model_endpoint_bring_your_own.ipynb

```diff
@@ -11,7 +11,7 @@
     "\n",
```

Comment:
<p>Add the following cell to install MXNet if not exist:</p><p> </p><pre class="ql-syntax" spellcheck="false">import pip

def import_or_install(package):
    try:
        __import__(package)
    except ImportError:
        pip.main(['install', package])  
        
import_or_install('mxnet')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>